### PR TITLE
Added the DropExtrema function

### PR DESCRIPTION
**Pull request by Jay-sanjay**
Issue  #193 fixing
Added the actual function for DropExtrema ,

### CODE REVIEW

File: src/TableTransforms.jl [hunk 1]
```diff
@@ -57,6 +57,7 @@ export
   Replace,
   Coalesce,
   Coerce,
+  dropExtrema,
```
Comment (juliohm) [inline src/TableTransforms.jl:60]: Notice that we have a naming convention that needs to be followed. The transforms always start with a capital letter.

File: src/transforms/dropExtrema.jl [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+        DropExtrema(low, high)
```
Comment (juliohm) [inline src/transforms/dropExtrema.jl:5]: Try to follow the docstring conventions `Scale` transform as it also has a low and a high.

File: src/transforms/dropExtrema.jl [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    
+    Applies the DropExtrema transform to the specified column `col` with interval [low, high].
+    
+    # Examples
+    ```julia
+    DropExtrema(0.01, 0.99)
+    DropExtrema(:column_name, 1.0, 10.0)
+    
+"""
+function drop_extrema(df::DataFrame, column_name::Symbol, low::Float64, high::Float64)
```
Comment (juliohm) [inline src/transforms/dropExtrema.jl:20]: You need to implement the same functions that are implemented for other transforms. See the developer guide for more information: https://juliaml.github.io/TableTransforms.jl/stable/devguide/

**Review by juliohm — CHANGES_REQUESTED** [review]

File: src/TableTransforms.jl (hunk 1 above)
Comment (Jay-sanjay) [reply src/TableTransforms.jl:60]: I will update it sir , thanks for pointing out

File: src/transforms/dropExtrema.jl (hunk 2 above)
Comment (Jay-sanjay) [reply src/transforms/dropExtrema.jl:5]: I will update it sir similar to what is done in other transform methods , thanks for pointing out.

File: src/transforms/dropExtrema.jl (hunk 3 above)
Comment (Jay-sanjay) [reply src/transforms/dropExtrema.jl:20]: yeah sir I will check it

**Comment by Jay-sanjay** [comment]
@juliohm  sir I am now working on the testCases part, which is according to me the potential reason for failing checks
Sir can you please tell what command to run in order to run the complete project locally in my pc.

File: src/transforms/dropExtrema.jl [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    col::Symbol
+end
+
+DropExtrema(low::Real, high::Real) = DropExtrema(low, high, :)
+DropExtrema(col::Symbol, low::Real, high::Real) = DropExtrema(low, high, col)
+
+function TransformsBase.apply(transform::DropExtrema, X)
+    a = quantile(X[!, transform.col], transform.low)
+    b = quantile(X[!, transform.col], transform.high)
+    filtered_rows = filter(row -> a <= row[!, transform.col] <= b, 1:nrow(X))
```
Comment (juliohm) [inline src/transforms/dropExtrema.jl:33]: take a look at the other transforms, we don't use DataFrame syntax, we need to use Tables.jl as an interface.

File: src/transforms/dropExtrema.jl [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+# Examples
+```julia
+DropExtrema(0.01, 0.99)
+DropExtrema(:column_name, 1.0, 10.0)
+```
+"""
+struct DropExtrema{T<:Real} <: TransformsBase.Transform
+    low::T
+    high::T
+    col::Symbol
```
Comment (juliohm) [inline src/transforms/dropExtrema.jl:24]: all other files use 2 whitespaces for indentation

File: src/transforms/dropExtrema.jl [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+end
+
+DropExtrema(low::Real, high::Real) = DropExtrema(low, high, :)
+DropExtrema(col::Symbol, low::Real, high::Real) = DropExtrema(low, high, col)
```
Comment (juliohm) [inline src/transforms/dropExtrema.jl:28]: these options should be kwargs instead of args

File: test/transforms.jl [hunk 7]
```diff
@@ -24,6 +24,7 @@ transformfiles = [
   "coltable.jl",
   "sequential.jl",
   "parallel.jl"
+  "dropExtrema.jl"
```
Comment (juliohm) [inline test/transforms.jl:27]: the name of source files should be lowercase

**Review by juliohm — CHANGES_REQUESTED** [review]
still lots to go. try to read the Julia documentation, the Tables.jl documentation and then come back here to contribute. In particular, try to ask in Zulip or other forums these basic user questions about tests, etc. people can help there as well.

here we can give you support about specifics of TableTransforms.jl

**Comment by juliohm** [comment]
Implemented in #201

**Pull request closed without merging**